framework/packages/abstract-client/src/client.rs

```diff
@@ -97,6 +120,94 @@ mod test_utils {
         }
     }
 
+    pub mod cw20_builder {
+        // Re-exports to limit dependencies for consumer.
+        pub use cw20::{msg::Cw20ExecuteMsgFns, *};
+        pub use cw20_base::msg::{InstantiateMarketingInfo, QueryMsgFns as Cw20QueryMsgFns};
+        pub use cw_plus_interface::cw20_base::Cw20Base;
+
+        use cosmwasm_std::Addr;
+
+        use cw_orch::prelude::{CwOrchInstantiate, CwOrchUpload, Mock};
+        use cw_plus_interface::cw20_base::InstantiateMsg;
+
+        use crate::client::AbstractClientResult;
+
+        pub struct Cw20Builder {
+            chain: Mock,
+            name: String,
+            symbol: String,
+            decimals: u8,
```

Comment:
I guess decimals could have default value as well

Reply:
Which default would make sense do you think?

Reply:
6 is the default value for Cosmos coins (and cw20s afaik)